### PR TITLE
Restrict parent imports, use @-imports instead

### PR DESCRIPTION
**Pull request by fregante**
- Fixes https://github.com/pixiebrix/pixiebrix-extension/issues/808

Error example:

```
'./actionPanel/ActionPanelApp' import is restricted from being used by a pattern. Use root-based imports (`import "@/something"`) instead of relative imports. [Error/no-restricted-imports]
```

I made this a warning for now because it finds 201 instances and they require manual intervention. Alternatively the PR can be left open until I fix them.

```
❯ nr lint -- --quiet --format unix
/Users/rico/Dev/pixie/pixiebrix-extension/src/__mocks__/webextDetectPage.ts:20
/Users/rico/Dev/pixie/pixiebrix-extension/src/action.tsx:23
/Users/rico/Dev/pixie/pixiebrix-extension/src/action.tsx:34
/Users/rico/Dev/pixie/pixiebrix-extension/src/background.ts:28
/Users/rico/Dev/pixie/pixiebrix-extension/src/background.ts:29
/Users/rico/Dev/pixie/pixiebrix-extension/src/background.ts:30
/Users/rico/Dev/pixie/pixiebrix-extension/src/background.ts:31
/Users/rico/Dev/pixie/pixiebrix-extension/src/background.ts:32
/Users/rico/Dev/pixie/pixiebrix-extension/src/background.ts:33
/Users/rico/Dev/pixie/pixiebrix-extension/src/background.ts:34
/Users/rico/Dev/pixie/pixiebrix-extension/src/background.ts:35
/Users/rico/Dev/pixie/pixiebrix-extension/src/background.ts:36
/Users/rico/Dev/pixie/pixiebrix-extension/src/background.ts:37
/Users/rico/Dev/pixie/pixiebrix-extension/src/background.ts:38
/Users/rico/Dev/pixie/pixiebrix-extension/src/background.ts:39
/Users/rico/Dev/pixie/pixiebrix-extension/src/background/dataStore.ts:20
/Users/rico/Dev/pixie/pixiebrix-extension/src/background/devtools/index.ts:18
/Users/rico/Dev/pixie/pixiebrix-extension/src/background/devtools/index.ts:19
/Users/rico/Dev/pixie/pixiebrix-extension/src/background/devtools/index.ts:20
/Users/rico/Dev/pixie/pixiebrix-extension/src/background/requests.test.ts:24
/Users/rico/Dev/pixie/pixiebrix-extension/src/blocks/combinators.ts:67
/Users/rico/Dev/pixie/pixiebrix-extension/src/blocks/effects/registerEffects.ts:19
/Users/rico/Dev/pixie/pixiebrix-extension/src/blocks/effects/registerEffects.ts:20
/Users/rico/Dev/pixie/pixiebrix-extension/src/blocks/effects/registerEffects.ts:21
/Users/rico/Dev/pixie/pixiebrix-extension/src/blocks/effects/registerEffects.ts:22
/Users/rico/Dev/pixie/pixiebrix-extension/src/blocks/effects/registerEffects.ts:23
/Users/rico/Dev/pixie/pixiebrix-extension/src/blocks/effects/registerEffects.ts:24
/Users/rico/Dev/pixie/pixiebrix-extension/src/blocks/effects/registerEffects.ts:25
/Users/rico/Dev/pixie/pixiebrix-extension/src/blocks/effects/registerEffects.ts:26
/Users/rico/Dev/pixie/pixiebrix-extension/src/blocks/effects/registerEffects.ts:27
/Users/rico/Dev/pixie/pixiebrix-extension/src/blocks/effects/registerEffects.ts:28
/Users/rico/Dev/pixie/pixiebrix-extension/src/blocks/effects/registerEffects.ts:29
/Users/rico/Dev/pixie/pixiebrix-extension/src/blocks/effects/registerEffects.ts:30
/Users/rico/Dev/pixie/pixiebrix-extension/src/blocks/effects/registerEffects.ts:31
/Users/rico/Dev/pixie/pixiebrix-extension/src/blocks/effects/registerEffects.ts:32
/Users/rico/Dev/pixie/pixiebrix-extension/src/blocks/effects/registerEffects.ts:33
/Users/rico/Dev/pixie/pixiebrix-extension/src/blocks/effects/registerEffects.ts:34
/Users/rico/Dev/pixie/pixiebrix-extension/src/blocks/effects/registerEffects.ts:35
/Users/rico/Dev/pixie/pixiebrix-extension/src/blocks/effects/registerEffects.ts:36
/Users/rico/Dev/pixie/pixiebrix-extension/src/blocks/effects/registerEffects.ts:37
/Users/rico/Dev/pixie/pixiebrix-extension/src/blocks/effects/registerEffects.ts:38
/Users/rico/Dev/pixie/pixiebrix-extension/src/blocks/readers/registerReaders.ts:19
/Users/rico/Dev/pixie/pixiebrix-extension/src/blocks/readers/registerReaders.ts:27
/Users/rico/Dev/pixie/pixiebrix-extension/src/blocks/readers/registerReaders.ts:28
/Users/rico/Dev/pixie/pixiebrix-extension/src/blocks/readers/registerReaders.ts:29
/Users/rico/Dev/pixie/pixiebrix-extension/src/blocks/readers/registerReaders.ts:30
/Users/rico/Dev/pixie/pixiebrix-extension/src/blocks/registerBuiltinBlocks.ts:18
/Users/rico/Dev/pixie/pixiebrix-extension/src/blocks/registerBuiltinBlocks.ts:19
/Users/rico/Dev/pixie/pixiebrix-extension/src/blocks/registerBuiltinBlocks.ts:20
/Users/rico/Dev/pixie/pixiebrix-extension/src/blocks/renderers/registerRenderers.ts:19
/Users/rico/Dev/pixie/pixiebrix-extension/src/blocks/renderers/registerRenderers.ts:20
/Users/rico/Dev/pixie/pixiebrix-extension/src/blocks/renderers/registerRenderers.ts:21
/Users/rico/Dev/pixie/pixiebrix-extension/src/blocks/renderers/registerRenderers.ts:22
/Users/rico/Dev/pixie/pixiebrix-extension/src/blocks/renderers/registerRenderers.ts:23
/Users/rico/Dev/pixie/pixiebrix-extension/src/blocks/renderers/registerRenderers.ts:24
/Users/rico/Dev/pixie/pixiebrix-extension/src/blocks/transformers/regex.test.ts:18
/Users/rico/Dev/pixie/pixiebrix-extension/src/blocks/transformers/registerTransformers.ts:20
/Users/rico/Dev/pixie/pixiebrix-extension/src/blocks/transformers/registerTransformers.ts:21
/Users/rico/Dev/pixie/pixiebrix-extension/src/blocks/transformers/registerTransformers.ts:22
/Users/rico/Dev/pixie/pixiebrix-extension/src/blocks/transformers/registerTransformers.ts:23
/Users/rico/Dev/pixie/pixiebrix-extension/src/blocks/transformers/registerTransformers.ts:24
/Users/rico/Dev/pixie/pixiebrix-extension/src/blocks/transformers/registerTransformers.ts:25
/Users/rico/Dev/pixie/pixiebrix-extension/src/blocks/transformers/registerTransformers.ts:26
/Users/rico/Dev/pixie/pixiebrix-extension/src/blocks/transformers/registerTransformers.ts:27
/Users/rico/Dev/pixie/pixiebrix-extension/src/blocks/transformers/registerTransformers.ts:28
/Users/rico/Dev/pixie/pixiebrix-extension/src/blocks/transformers/registerTransformers.ts:29
/Users/rico/Dev/pixie/pixiebrix-extension/src/blocks/transformers/registerTransformers.ts:30
/Users/rico/Dev/pixie/pixiebrix-extension/src/blocks/transformers/registerTransformers.ts:31
/Users/rico/Dev/pixie/pixiebrix-extension/src/blocks/transformers/registerTransformers.ts:32
/Users/rico/Dev/pixie/pixiebrix-extension/src/blocks/transformers/registerTransformers.ts:33
/Users/rico/Dev/pixie/pixiebrix-extension/src/blocks/transformers/registerTransformers.ts:34
/Users/rico/Dev/pixie/pixiebrix-extension/src/blocks/util.test.ts:18
/Users/rico/Dev/pixie/pixiebrix-extension/src/chrome.ts:22
/Users/rico/Dev/pixie/pixiebrix-extension/src/common.ts:18
/Users/rico/Dev/pixie/pixiebrix-extension/src/common.ts:19
/Users/rico/Dev/pixie/pixiebrix-extension/src/components/ChatWidget.tsx:20
/Users/rico/Dev/pixie/pixiebrix-extension/src/components/ellipsisMenu/EllipsisMenu.stories.tsx:20
/Users/rico/Dev/pixie/pixiebrix-extension/src/components/ellipsisMenu/EllipsisMenu.tsx:22
/Users/rico/Dev/pixie/pixiebrix-extension/src/components/fields/BlockField.tsx:19
/Users/rico/Dev/pixie/pixiebrix-extension/src/components/fields/BlockField.tsx:40
/Users/rico/Dev/pixie/pixiebrix-extension/src/components/fields/BlockModal.tsx:49
/Users/rico/Dev/pixie/pixiebrix-extension/src/components/fields/BlockModal.tsx:55
/Users/rico/Dev/pixie/pixiebrix-extension/src/components/fields/ServiceModal.tsx:34
/Users/rico/Dev/pixie/pixiebrix-extension/src/components/fields/ServiceModal.tsx:37
/Users/rico/Dev/pixie/pixiebrix-extension/src/components/schemaTree/SchemaTree.tsx:24
/Users/rico/Dev/pixie/pixiebrix-extension/src/contentScript/executor.ts:29
/Users/rico/Dev/pixie/pixiebrix-extension/src/contrib/automationanywhere/index.ts:18
/Users/rico/Dev/pixie/pixiebrix-extension/src/contrib/google/bigquery/query.ts:21
/Users/rico/Dev/pixie/pixiebrix-extension/src/contrib/google/index.ts:18
/Users/rico/Dev/pixie/pixiebrix-extension/src/contrib/google/index.ts:19
/Users/rico/Dev/pixie/pixiebrix-extension/src/contrib/google/index.ts:20
/Users/rico/Dev/pixie/pixiebrix-extension/src/contrib/google/initGoogle.ts:20
/Users/rico/Dev/pixie/pixiebrix-extension/src/contrib/google/initGoogle.ts:21
/Users/rico/Dev/pixie/pixiebrix-extension/src/contrib/hubspot/index.ts:18
/Users/rico/Dev/pixie/pixiebrix-extension/src/contrib/pipedrive/index.ts:18
/Users/rico/Dev/pixie/pixiebrix-extension/src/contrib/pipedrive/index.ts:19
/Users/rico/Dev/pixie/pixiebrix-extension/src/contrib/pipedrive/index.ts:20
/Users/rico/Dev/pixie/pixiebrix-extension/src/contrib/registerContribBlocks.ts:19
/Users/rico/Dev/pixie/pixiebrix-extension/src/contrib/registerContribBlocks.ts:20
/Users/rico/Dev/pixie/pixiebrix-extension/src/contrib/registerContribBlocks.ts:28
/Users/rico/Dev/pixie/pixiebrix-extension/src/contrib/registerContribBlocks.ts:33
/Users/rico/Dev/pixie/pixiebrix-extension/src/contrib/registerContribBlocks.ts:34
/Users/rico/Dev/pixie/pixiebrix-extension/src/contrib/registerContribBlocks.ts:35
/Users/rico/Dev/pixie/pixiebrix-extension/src/contrib/registerContribBlocks.ts:36
/Users/rico/Dev/pixie/pixiebrix-extension/src/contrib/registerContribBlocks.ts:37
/Users/rico/Dev/pixie/pixiebrix-extension/src/contrib/salesforce/index.ts:18
/Users/rico/Dev/pixie/pixiebrix-extension/src/contrib/slack/index.ts:18
/Users/rico/Dev/pixie/pixiebrix-extension/src/contrib/uipath/index.ts:18
/Users/rico/Dev/pixie/pixiebrix-extension/src/contrib/uipath/index.ts:19
/Users/rico/Dev/pixie/pixiebrix-extension/src/contrib/uipath/index.ts:20
/Users/rico/Dev/pixie/pixiebrix-extension/src/contrib/zapier/index.ts:18
/Users/rico/Dev/pixie/pixiebrix-extension/src/devTools/Panel.tsx:25
/Users/rico/Dev/pixie/pixiebrix-extension/src/devTools/editor/fields/Fields.tsx:24
/Users/rico/Dev/pixie/pixiebrix-extension/src/devTools/editor/fields/creatableAutosuggest/CreatableAutosuggest.tsx:19
/Users/rico/Dev/pixie/pixiebrix-extension/src/devTools/editor/fields/selectorListItem/SelectorListItem.stories.tsx:20
/Users/rico/Dev/pixie/pixiebrix-extension/src/devTools/editor/fields/selectorListItem/SelectorListItem.tsx:19
/Users/rico/Dev/pixie/pixiebrix-extension/src/devTools/editor/tabs/EffectTab.tsx:23
/Users/rico/Dev/pixie/pixiebrix-extension/src/devTools/editor/tabs/LogsTab.tsx:21
/Users/rico/Dev/pixie/pixiebrix-extension/src/devTools/editor/tabs/ServicesTab.tsx:22
/Users/rico/Dev/pixie/pixiebrix-extension/src/devTools/editor/tabs/ServicesTab.tsx:34
/Users/rico/Dev/pixie/pixiebrix-extension/src/devtools.ts:24
/Users/rico/Dev/pixie/pixiebrix-extension/src/frameworks/adapters.ts:19
/Users/rico/Dev/pixie/pixiebrix-extension/src/frameworks/adapters.ts:21
/Users/rico/Dev/pixie/pixiebrix-extension/src/frameworks/adapters.ts:22
/Users/rico/Dev/pixie/pixiebrix-extension/src/hooks/auth.ts:23
/Users/rico/Dev/pixie/pixiebrix-extension/src/hooks/auth.ts:27
/Users/rico/Dev/pixie/pixiebrix-extension/src/icons/IconSelector.tsx:26
/Users/rico/Dev/pixie/pixiebrix-extension/src/layout/Banner.tsx:20
/Users/rico/Dev/pixie/pixiebrix-extension/src/layout/Navbar.tsx:37
/Users/rico/Dev/pixie/pixiebrix-extension/src/layout/Sidebar.tsx:32
/Users/rico/Dev/pixie/pixiebrix-extension/src/layout/Sidebar.tsx:33
/Users/rico/Dev/pixie/pixiebrix-extension/src/layout/SidebarLink.tsx:25
/Users/rico/Dev/pixie/pixiebrix-extension/src/nativeEditor/index.ts:18
/Users/rico/Dev/pixie/pixiebrix-extension/src/nativeEditor/index.ts:26
/Users/rico/Dev/pixie/pixiebrix-extension/src/nativeEditor/index.ts:27
/Users/rico/Dev/pixie/pixiebrix-extension/src/nativeEditor/insertButton.tsx:22
/Users/rico/Dev/pixie/pixiebrix-extension/src/nativeEditor/insertButton.tsx:23
/Users/rico/Dev/pixie/pixiebrix-extension/src/nativeEditor/insertButton.tsx:25
/Users/rico/Dev/pixie/pixiebrix-extension/src/nativeEditor/insertButton.tsx:31
/Users/rico/Dev/pixie/pixiebrix-extension/src/nativeEditor/insertPanel.tsx:22
/Users/rico/Dev/pixie/pixiebrix-extension/src/nativeEditor/insertPanel.tsx:23
/Users/rico/Dev/pixie/pixiebrix-extension/src/nativeEditor/insertPanel.tsx:25
/Users/rico/Dev/pixie/pixiebrix-extension/src/options/App.tsx:19
/Users/rico/Dev/pixie/pixiebrix-extension/src/options/App.tsx:58
/Users/rico/Dev/pixie/pixiebrix-extension/src/options/pages/SetupPage.tsx:30
/Users/rico/Dev/pixie/pixiebrix-extension/src/options/pages/brickEditor/CreatePage.tsx:23
/Users/rico/Dev/pixie/pixiebrix-extension/src/options/pages/brickEditor/CreatePage.tsx:24
/Users/rico/Dev/pixie/pixiebrix-extension/src/options/pages/brickEditor/EditPage.tsx:24
/Users/rico/Dev/pixie/pixiebrix-extension/src/options/pages/brickEditor/EditPage.tsx:27
/Users/rico/Dev/pixie/pixiebrix-extension/src/options/pages/brickEditor/EditPage.tsx:30
/Users/rico/Dev/pixie/pixiebrix-extension/src/options/pages/brickEditor/Editor.tsx:29
/Users/rico/Dev/pixie/pixiebrix-extension/src/options/pages/brickEditor/Editor.tsx:30
/Users/rico/Dev/pixie/pixiebrix-extension/src/options/pages/brickEditor/Editor.tsx:44
/Users/rico/Dev/pixie/pixiebrix-extension/src/options/pages/brickEditor/referenceTab/BlockResult.tsx:25
/Users/rico/Dev/pixie/pixiebrix-extension/src/options/pages/brickEditor/referenceTab/BlockResult.tsx:26
/Users/rico/Dev/pixie/pixiebrix-extension/src/options/pages/brickEditor/referenceTab/BrickDetail.stories.tsx:20
/Users/rico/Dev/pixie/pixiebrix-extension/src/options/pages/brickEditor/referenceTab/BrickDetail.test.tsx:20
/Users/rico/Dev/pixie/pixiebrix-extension/src/options/pages/brickEditor/referenceTab/BrickDetail.test.tsx:22
/Users/rico/Dev/pixie/pixiebrix-extension/src/options/pages/brickEditor/referenceTab/BrickDetail.tsx:27
/Users/rico/Dev/pixie/pixiebrix-extension/src/options/pages/brickEditor/referenceTab/BrickDetail.tsx:28
/Users/rico/Dev/pixie/pixiebrix-extension/src/options/pages/brickEditor/referenceTab/BrickDetail.tsx:30
/Users/rico/Dev/pixie/pixiebrix-extension/src/options/pages/brickEditor/referenceTab/BrickReference.tsx:30
/Users/rico/Dev/pixie/pixiebrix-extension/src/options/pages/brickEditor/referenceTab/BrickReference.tsx:32
/Users/rico/Dev/pixie/pixiebrix-extension/src/options/pages/brickEditor/referenceTab/BrickReference.tsx:33
/Users/rico/Dev/pixie/pixiebrix-extension/src/options/pages/brickEditor/referenceTab/BrickReference.tsx:34
/Users/rico/Dev/pixie/pixiebrix-extension/src/options/pages/brickEditor/useSubmitBrick.tsx:24
/Users/rico/Dev/pixie/pixiebrix-extension/src/options/pages/brickEditor/useSubmitBrick.tsx:25
/Users/rico/Dev/pixie/pixiebrix-extension/src/options/pages/installed/ActiveBricksCard.tsx:20
/Users/rico/Dev/pixie/pixiebrix-extension/src/options/pages/installed/ActiveBricksCard.tsx:22
/Users/rico/Dev/pixie/pixiebrix-extension/src/options/pages/installed/ActiveBricksCard.tsx:23
/Users/rico/Dev/pixie/pixiebrix-extension/src/options/pages/installed/ActiveBricksCard.tsx:25
/Users/rico/Dev/pixie/pixiebrix-extension/src/options/pages/installed/ExtensionGroup.tsx:37
/Users/rico/Dev/pixie/pixiebrix-extension/src/options/pages/installed/ExtensionGroup.tsx:38
/Users/rico/Dev/pixie/pixiebrix-extension/src/options/pages/installed/ExtensionGroupHeader.tsx:19
/Users/rico/Dev/pixie/pixiebrix-extension/src/options/pages/installed/ExtensionRows.tsx:20
/Users/rico/Dev/pixie/pixiebrix-extension/src/options/pages/installed/ExtensionRows.tsx:21
/Users/rico/Dev/pixie/pixiebrix-extension/src/options/pages/installed/ExtensionRows.tsx:22
/Users/rico/Dev/pixie/pixiebrix-extension/src/options/pages/installed/InstalledExtensionRow.tsx:37
/Users/rico/Dev/pixie/pixiebrix-extension/src/options/pages/installed/InstalledPage.tsx:26
/Users/rico/Dev/pixie/pixiebrix-extension/src/options/pages/installed/InstalledPage.tsx:43
/Users/rico/Dev/pixie/pixiebrix-extension/src/options/pages/marketplace/ActivateWizard.tsx:22
/Users/rico/Dev/pixie/pixiebrix-extension/src/options/pages/marketplace/ActivateWizard.tsx:24
/Users/rico/Dev/pixie/pixiebrix-extension/src/options/pages/marketplace/ActivateWizard.tsx:28
/Users/rico/Dev/pixie/pixiebrix-extension/src/options/pages/marketplace/ActivateWizard.tsx:29
/Users/rico/Dev/pixie/pixiebrix-extension/src/options/pages/services/PrivateServicesCard.tsx:22
/Users/rico/Dev/pixie/pixiebrix-extension/src/options/pages/services/ServicesEditor.tsx:20
/Users/rico/Dev/pixie/pixiebrix-extension/src/options/pages/services/ServicesEditor.tsx:24
/Users/rico/Dev/pixie/pixiebrix-extension/src/options/pages/services/ServicesEditor.tsx:25
/Users/rico/Dev/pixie/pixiebrix-extension/src/options/pages/services/ServicesEditor.tsx:26
/Users/rico/Dev/pixie/pixiebrix-extension/src/options/pages/services/ServicesEditor.tsx:27
/Users/rico/Dev/pixie/pixiebrix-extension/src/options/pages/services/ServicesEditor.tsx:29
/Users/rico/Dev/pixie/pixiebrix-extension/src/options/pages/services/ServicesEditor.tsx:30
/Users/rico/Dev/pixie/pixiebrix-extension/src/options/pages/templates/TemplatesPage.tsx:47
/Users/rico/Dev/pixie/pixiebrix-extension/src/options/pages/workshop/WorkshopPage.tsx:57
/Users/rico/Dev/pixie/pixiebrix-extension/src/options/store.ts:21
/Users/rico/Dev/pixie/pixiebrix-extension/src/pages/marketplace/MarketplacePage.tsx:26
/Users/rico/Dev/pixie/pixiebrix-extension/src/script.ts:47
/Users/rico/Dev/pixie/pixiebrix-extension/src/script.ts:68
/Users/rico/Dev/pixie/pixiebrix-extension/src/types.ts:18
/Users/rico/Dev/pixie/pixiebrix-extension/src/utils/objToYaml.test.ts:18
/Users/rico/Dev/pixie/pixiebrix-extension/src/validators/generic.ts:51

201 problems
```

### CODE REVIEW

**Comment by BALEHOK** [comment]
First, let me say @-based imports are nice.
Second, in my opinion, they are nice but not always.

Suppose I work on `LogTable`.
It imports `LogEntry`, `EntryRow` and `styles`.

```
import type { LogEntry } from "@/background/logging";
import EntryRow from "@/components/logViewer/EntryRow";
import styles from "@/components/logViewer/LogTable.module.scss";
```
vs
```
import type { LogEntry } from "@/background/logging";
import EntryRow from "./EntryRow";
import styles from "./LogTable.module.scss";
```

`LogEntry` imported with `@` makes sense to me. A component from another domain. With the navigation of several directories up the hierarchy, the import path looks crazy.
However for the latter two imports, `EntryRow` and `styles`, I'd prefer a relative path (short, same domain).

To sum up, my preference would be using `@` if you need to go up the hierarchy (`../../../components/Component`), otherwise use relative import.

**Comment by fregante** [comment]
Yeah it's mostly about consistency and ease of grepping, but it's not a strong preference either way.

I agree that `../` can be avoided and there's a specific rule for that if we don't end up enforcing `@/`

https://github.com/import-js/eslint-plugin-import/blob/main/docs/rules/no-relative-parent-imports.md

**Comment by BLoe** [comment]
Also with CSS modules I've been importing them like:
```
import styles from "./EditorNode.module.scss";
```

So this would break the @-only rule as well.

**Comment by BLoe** [comment]
I agree it would be kind of annoying to have to use @ imports for CSS modules in the same folder.

**Comment by BALEHOK** [comment]
I guess we need a decision here?

My preferences (in order of how much I like them):
1. `@` for parent imports, relative otherwise;
2. `@` always
3. relaxed, no rule regarding imports at all

I'm fine with any of those.

Looks like @BLoe doesn't favor option 2 (full path for `styles`), hence we end up with 1 or 3.

Since @fregante has no strong preference but is for consistency (which is not about option 3), let's go with 1!?

cc @twschiller

**Comment by fregante** [comment]
1 is fine for me, even though the second part is not enforceable (there's no rule to enforce `./` because that's how it's normally done in the first place)

File: .eslintrc [hunk 1]
```diff
@@ -46,6 +46,10 @@
           {
             "group": ["react-bootstrap/*", "!react-bootstrap/types"],
             "message": "You can import \"react-bootstrap\" instead of \"react-bootstrap/*\"."
+          },
+          {
+            "group": ["../*", "./*"],
```
Comment (BALEHOK) [inline .eslintrc:51]: As per discussion we keep "../*" and remove "./*".

**Comment by fregante** [comment]
I tried using `import/no-relative-parent-imports` but it seems that it resolves `@/` imports and understands that some imports are of parents, rather than just looking for the `../` string like `no-restricted-imports` does.

`@`-based imports seem to be frowned upon in the `import` plugin so there's no specific rule for that either: 

- https://github.com/import-js/eslint-plugin-import/issues/1420

Also worth keeping track of:

- https://github.com/import-js/eslint-plugin-import/issues/669

**Comment by BALEHOK** [comment]
> I tried using `import/no-relative-parent-imports` but it seems that it resolves `@/` imports and understands that some imports are of parents, rather than just looking for the `../` string like `no-restricted-imports` does.
> 
> `@`-based imports seem to be frowned upon in the `import` plugin so there's no specific rule for that either:
> 
> * [Enforce imports paths from project's root import-js/eslint-plugin-import#1420](https://github.com/import-js/eslint-plugin-import/issues/1420)
> 
> Also worth keeping track of:
> 
> * [Rule proposal: no-relative-path import-js/eslint-plugin-import#669](https://github.com/import-js/eslint-plugin-import/issues/669)

I'm a little bit confused. Does the rule work as we want it to (as the [comment](https://github.com/import-js/eslint-plugin-import/issues/669#issuecomment-316438608) suggests)?

**Comment by fregante** [comment]
import/no-relative-parent-imports doesn't work for us;

no-restricted-imports does, so I set it up here.

**Comment by twschiller** [comment]
@fregante is this PR still relevant? Or should this go in the shared config?

File: package.json [hunk 2]
```diff
@@ -199,7 +199,7 @@
     "css-minimizer-webpack-plugin": "^3.0.2",
     "dotenv": "^10.0.0",
     "eslint": "^7.30.0",
-    "eslint-config-pixiebrix": "^0.0.1",
+    "eslint-config-pixiebrix": "^0.9.0",
```
Comment (fregante) [inline package.json:202]: Moved to the shared config: https://github.com/fregante/eslint-config-pixiebrix/blob/0f30e8c38253766c71b9f6aedc000120657c29f2/index.js#L54

Generally it's still ok to add rules locally and then move them to the shared config later, but I thought I'd just do it now in this case.

**Review by twschiller — APPROVED** [review]
